product_secondary_unit/tests/test_secondary_unit_mixin.py

```diff
@@ -60,7 +60,7 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         cls.loader.restore_registry()
-        super(TestProductSecondaryUnitMixin, cls).tearDownClass()
+        return super(TestProductSecondaryUnitMixin, cls).tearDownClass()
```

Comment:
Mmmh, not sure return is mandatory

Reply:
I don't think so:
```
@classmethod
def tearDownClass(cls):
      "Hook method for deconstructing the class fixture after running all tests in the class."
```

This is the definition on python 3.7.

Reply:
@remihb could you remove this return so the module can be merged?
Thanks,

@rousseldenis FYI (thanks for the review :) )

Reply:
Done.
But pre-commit fails without return

Reply:
True
`product_secondary_unit/tests/test_secondary_unit_mixin.py:61: [W8110(missing-return), TestProductSecondaryUnitMixin.tearDownClass] Missing `return` (`super` is used) in method tearDownClass.`

What do you suggest @rousseldenis ?

I'd keep the return